src/actions/sessionActions.js

```diff
@@ -5,14 +5,25 @@ import sessionApi from '../api/sessionApi';
 
 export const login = user =>
   () =>
-    sessionApi.login({ user }).then(({ user }) => {
+    sessionApi.login({ user }).then(({ data: user }) => {
       sessionService.saveUser(user);
     }).catch((err) => {
       throw new SubmissionError({
         _error: err.errors
       });
     });
 
+export const loginFacebook = user =>
+  () => {
+    sessionApi.login({ access_token: user }).then(({ data: user }) => {
```

Comment:
No need to use snake case. You just use camel case and then in `apiService.js` the keys get converted to snake via the `humps` package.